File: python_draft/src/java/type2/ProcessDataLiterals.java
public class ProcessDataLiterals {
public static java.util.List<Integer> processDataV2(java.util.List<Integer> input) {
    java.util.List<Integer> result = new java.util.ArrayList<>();
    for (int item : input) {
        if (item % 4 == 0) {
            result.add(item * 3);
        }
    }
    return result;
}

}

File: python_draft/src/cpp/type2/process_data_literals.cpp
#include <vector>
std::vector<int> processDataV2(const std::vector<int>& input){
    std::vector<int> result;
    for(int item : input){
        if(item % 4 == 0) result.push_back(item * 3);
    }
    return result;
}
